### DISCUSSION TITLE
How to export to HTML using Python in cadwork?

### DISCUSSION THREAD

**Discussion by lucas-ita** [cwapi3d/cwapi3dpython#163] [Q&A] [answered]
Is there any method I can use to export Cadwork elements to HTML?
something like: `export_html(element_id, filePath)`

**Comment by Brunner246** [comment, marked as answer]
why not using `export_webgl(element_id_list: List[int], file_path: str)` in `file_controller` ?

**Reply by lucas-ita** [reply]
Thanks!
